test/jdk/tools/jpackage/share/ErrorTest.java

```diff
@@ -88,6 +90,31 @@ enum Token {
 
             return appImageCmd.outputBundle().toString();
         }),
+        INVALID_JDK_BUNDLE(cmd -> {
+            // Missing "Contents/MacOS/libjli.dylib"
+            try {
+                final Path runtimePath = TKit.createTempDirectory("invalidJDKBundle");
+                Files.createDirectories(runtimePath.resolve("Contents/Home"));
+                Files.createFile(runtimePath.resolve("Contents/Info.plist"));
+                return runtimePath.toAbsolutePath().toString();
+            } catch (IOException ex) {
+                TKit.error(ex.getMessage());
+                return null;
```

Comment:
What is the point of this exception "handling"? What problem does it solve? What is the caller supposed to do with the `null`?